apps/ehr/package.json

```
@@ -40,7 +40,7 @@
     "test": "npm run unit-tests -- --watch=false && npm run component-tests -- --watch=false",
     "e2e:login": "npm run e2e-skeleton ./tests/e2e/login",
     "e2e:specs": "npm run e2e-skeleton ./tests/e2e/specs",
-    "e2e:specs:ui": "npm run e2e-skeleton ./tests/e2e/specs -- --ui",
+    "e2e:specs:ui": "npm run e2e-skeleton ./tests/e2e/specs",
```

i think this was only used for testing purposes. revert this please


```suggestion
    "e2e:specs:ui": "npm run e2e-skeleton ./tests/e2e/specs -- --ui",
```